src/browser/atom-portable.coffee

```diff
@@ -1,6 +1,6 @@
 fs = require 'fs-plus'
 path = require 'path'
-ipc = require 'ipc'
+ipc = require 'ipc-main'
```

Comment:
And...switching it back to ipc-renderer makes it work again.  Really hope there isn't a race condition of some sort going on here.